sorcery.gemspec

```diff
@@ -38,7 +38,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'oauth2', '~> 2.0'
 
   s.add_development_dependency 'byebug', '~> 10.0.0'
-  s.add_development_dependency 'rspec-rails', '~> 3.7.0'
```

Comment:
Might be better to just remove the version lock instead, but I vaguely recall dev dependencies in the gemspec being deprecated anyway.